pkg/scheduler/objects/queue.go

```diff
@@ -1220,6 +1220,63 @@ func (sq *Queue) GetMaxResource() *resources.Resource {
 	return sq.internalGetMax(limit)
 }
 
+// Starting with the root, descend down to the target queue allowing children to override Resource values .
+// If the root includes an explicit 0 value for a Resource, do not include it in the accumulator and treat it as missing.
+// If no children provide a maximum capacity override, the resulting value will be the value found on the Root.
```

Comment:
Add the function name as the first word in the description per godoc conventions. I.e. "GetFairMaxResource computes the fair maximum resources ...."